source/UF/Size-TruncatedConnected.lagda

```diff
@@ -0,0 +1,413 @@
+  pr X (is-prop'-implies-is-prop X-prop)
+ Lemma-2-3[being-truncated-gives-local-smallness] {_} {_} {succ n} pr X-trunc x x' =
+  Lemma-2-3[being-truncated-gives-local-smallness] pr (X-trunc x x')
+
+ being-truncated-gives-local-smallness =
+  Lemma-2-3[being-truncated-gives-local-smallness]
+
+\end{code}
+
+We note that Lemma-2-3 provides one side of a bimplication involving
+propositional resizing. We will now record the other direction.
+
+\begin{code}
+
+ being-truncated-gives-local-smallness-implies-resizing
+  : ({X : 𝓤 ̇} {n : ℕ₋₂} → X is (n + 1) truncated → X is ι (n + 2) locally-small)
+  → propositional-resizing 𝓤 𝓥
+ being-truncated-gives-local-smallness-implies-resizing
+  trunc-gives-loc-small P P-prop =
+   trunc-gives-loc-small {P} {−2} (is-prop-implies-is-prop' P-prop)
+
+\end{code}
+
+Lemma 2.4. of [1]
+
+\begin{code}
+
+ Lemma-2-4[type-with-truncated-map-to-locally-small-type-is-locally-small]
+  : {X : 𝓤 ̇} {Y : 𝓦 ̇} {f : X → Y} {n : ℕ₋₂}
+  → Propositional-Resizing
+  → f is (n + 1) truncated-map
+  → Y is ι (n + 2) locally-small
+  → X is ι (n + 2) locally-small
+ Lemma-2-4[type-with-truncated-map-to-locally-small-type-is-locally-small]
+  {_} {_} {_} {_} {f} {_} pr f-trunc Y-loc-small =
+  local-smallness-is-closed-under-≃ (total-fiber-is-domain f)
+   (local-smallness-is-closed-under-Σ Y-loc-small
+    (λ y → Lemma-2-3[being-truncated-gives-local-smallness] pr (f-trunc y)))
+
+ type-with-truncated-map-to-locally-small-type-is-locally-small =
+  Lemma-2-4[type-with-truncated-map-to-locally-small-type-is-locally-small]
+
+\end{code}
+
+Lemma 2.5. of [1]
+
+\begin{code}
+
+ Lemma-2-5[connected-type-with-truncated-map-to-locally-small-type-is-small]
+  : {X : 𝓤 ̇} {Y : 𝓦 ̇} {f : X → Y} {n : ℕ₋₂}
+  → Univalence
+  → Join-Construction-Result {𝓤} {𝓤}
+  → Propositional-Resizing
+  → f is (n + 1) truncated-map
+  → Y is ι (n + 2) locally-small
+  → X is (n + 1) connected
+  → X is 𝓥 small
+ Lemma-2-5[connected-type-with-truncated-map-to-locally-small-type-is-small]
+  {𝓤} {_} {X} {_} {_} {n} ua j pr f-trunc Y-loc-small X-conn =
+  ∥∥-rec (being-small-is-prop ua X 𝓥) VI (center II)
+  where
+   I : X is ι (n + 2) locally-small
+   I = Lemma-2-4[type-with-truncated-map-to-locally-small-type-is-locally-small]
+        pr f-trunc Y-loc-small
+   II : X is −1 connected
+   II = connectedness-is-lower-closed' ⋆ X-conn
+   III : X → 𝟙 {𝓤} → X
+   III x ⋆ = x
+   IV : (x : X) → (III x) is n connected-map
+   IV x = basepoint-map-is-less-connected (ua _) (III x) X-conn
+   V : 𝟙 {𝓤} is 𝓥 small
+   V = pr 𝟙 𝟙-is-prop
+   VI : X → X is 𝓥 small
+   VI x = Prop-2-2[locally-small-type-with-connected-map-from-small-type-is-small]
+           ua j (IV x) V I
+
+ connected-type-with-truncated-map-to-locally-small-type-is-small =
+  Lemma-2-5[connected-type-with-truncated-map-to-locally-small-type-is-small]
+
+\end{code}
+
+In [1] Christense proves Theorem 2.6. with propositional resizing but later found
```

Comment:
`Christense[n]`

Reply:
I think this comment is outdated now as the proof in the published version of [1] actually avoids propositional resizing.

Reply:
He first proves it with resizing then without

Reply:
Then the comment should still be updated to reflect that as it kind of suggests that [1] does not contain the new proof.